src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/InputFormattedJavaDocTagContinuationIndentation.java

```diff
@@ -0,0 +1,306 @@
+package com.google.checkstyle.test.chapter7javadoc.rule713atclauses;
+
+import java.io.Serializable;
+
+/**
+ * Some javadoc.
+ *
+ * @since Some javadoc. Some javadoc.
+ * @version 1.0
+ * @deprecated Some javadoc. Some javadoc.
+ * @see Some javadoc.
+ * @author max Some javadoc.
+ */
+class InputFormattedJavaDocTagContinuationIndentation implements Serializable {
+  /**
+   * The client's first name.
+   *
+   * @serial Some javadoc. Some javadoc.
+   */
+  private String firstName;
+
+  /**
+   * The client's first name.
+   *
+   * @serial Some javadoc.
+   */
+  private String secondName;
+
+  /**
+   * The client's first name.
+   *
+   * @serialField Some javadoc.
+   */
+  private String thirdName;
+
+  // 7 violations 4 lines above:
+  //  'Javadoc comment at column 30 has parse error. .* expecting <EOF> while parsing JAVADOC'
+  //  'Javadoc comment at column 30 has parse error. .* expecting <EOF> while parsing JAVADOC'
+  //  'Javadoc comment at column 30 has parse error. .* expecting <EOF> while parsing JAVADOC'
+  //  'Javadoc comment at column 30 has parse error. .* expecting <EOF> while parsing JAVADOC'
+  //  'Javadoc comment at column 30 has parse error. .* expecting <EOF> while parsing JAVADOC'
+  //  'Javadoc comment at column 30 has parse error. .* expecting <EOF> while parsing JAVADOC'
+  //  'Javadoc comment at column 30 has parse error. .* expecting <EOF> while parsing JAVADOC'
```

Comment:
after formatting, our test started giving this violation, not sure what is causing it to occur

Reply:
this is parser error, it means Checkstyle think this javadoc is not valid at all.

please run this java doc by `javadoc` tool of jdk to see if it generate html correctly or also fail with error.
if `javadoc` tool also give error - it is defect of formatter.
if `javadoc` tool generates html content - it is defect in Checkstyle parser of javadoc, issue should be created on us.

Reply:
as issue is created  (or us or formatter, PR can be merged)

Reply:
if issue is on us, make Input code as:

```
// until #xxxx
// 7 violations 5 lines above:
```

Reply:
https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#tag

> `@serial (or @serialField or @serialData)`

https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#@serial
no text/description should be used for this tag, it is syntax error of javadoc.

```
rivanov@p5510:/var/tmp$ cat Test.java 
public class Test {
  /**
   * The client's first name.
   *
   * @serialField some text.
   */
  public String thirdName;
}
rivanov@p5510:/var/tmp$ javadoc Test.java 
Loading source file Test.java...
Constructing Javadoc information...
Standard Doclet version 11.0.14.1
Building tree for all the packages and classes...
Generating ./Test.html...
Test.java:5: error: syntax error in reference
   * @serialField some text.
     ^
Generating ./package-summary.html...
Generating ./package-tree.html...
Generating ./constant-values.html...
Building index for all the packages and classes...
Generating ./overview-tree.html...
Generating ./index-all.html...
Building index for all classes...
Generating ./allclasses-index.html...
Generating ./allpackages-index.html...
Generating ./deprecated-list.html...
Building index for all classes...
Generating ./allclasses.html...
Generating ./allclasses.html...
Generating ./index.html...
Generating ./help-doc.html...
1 error

``` 

error appears only on elements that are about to appear in html , I made field  `public` to show a problem.

Reply:
correct syntax:
```
rivanov@p5510:/var/tmp$ cat Test.java 
public class Test {
  /**
   * The client's first name.
   *
   * @serialField serial String
   *              Path components separated by separator.
   */
  public String thirdName;
  private String serial;
}
rivanov@p5510:/var/tmp$ javadoc Test.java 
Loading source file Test.java...
Constructing Javadoc information...
Standard Doclet version 11.0.14.1
Building tree for all the packages and classes...
Generating ./Test.html...
Generating ./package-summary.html...
Generating ./package-tree.html...
Generating ./constant-values.html...
Building index for all the packages and classes...
Generating ./overview-tree.html...
Generating ./index-all.html...
Building index for all classes...
Generating ./allclasses-index.html...
Generating ./allpackages-index.html...
Generating ./deprecated-list.html...
Building index for all classes...
Generating ./allclasses.html...
Generating ./allclasses.html...
Generating ./index.html...
Generating ./help-doc.html...

```

Reply:
tag without  identifier/description is not valid
```
rivanov@p5510:/var/tmp$ cat Test.java 
public class Test {
  /**
   * The client's first name.
   *
   * @serialField
   */
  public String thirdName;
  private String serial;
}
rivanov@p5510:/var/tmp$ javadoc Test.java 
Loading source file Test.java...
Constructing Javadoc information...
Standard Doclet version 11.0.14.1
Building tree for all the packages and classes...
Generating ./Test.html...
Test.java:5: error: identifier expected
   * @serialField
     ^
Generating ./package-summary.html...
Generating ./package-tree.html...
Generating ./constant-values.html...
Building index for all the packages and classes...
Generating ./overview-tree.html...
Generating ./index-all.html...
Building index for all classes...
Generating ./allclasses-index.html...
Generating ./allpackages-index.html...
Generating ./deprecated-list.html...
Building index for all classes...
Generating ./allclasses.html...
Generating ./allclasses.html...
Generating ./index.html...
Generating ./help-doc.html...
1 error
```

Reply:
I moved this case to https://github.com/checkstyle/checkstyle/issues/15340 , we already have issue, it is big limitation of Checkstyle.

I did workaround in Input.